assume/__init__.py

```diff
@@ -3,11 +3,6 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
 from assume.common import MarketConfig, MarketProduct
-from assume.common.scenario_loader import (
-    load_custom_units,
-    load_scenario_folder,
-    run_learning,
-)
 from assume.world import World
 
 __version__ = "0.0.1"
```

Comment:
not relevant to this, but general question, do we also need to update the version here as well?

Reply:
Actually this version is not needed at all, but is a common quasi-standard:
https://stackoverflow.com/a/459185

The way I manage this in other packages is by using:
`from importlib.metadata import version`
sets the value from the pyproject.toml metadata (which is what we want, without duplicating version information.

I will create PR for this